### PR TITLE
Win_copy: doc fix when using become

### PR DESCRIPTION
**Pull request by ShachafGoldstein**
##### SUMMARY
To handle cases such as #48577 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
win_copy

### CODE REVIEW

File: lib/ansible/modules/windows/win_copy.py [hunk 1]
```diff
@@ -91,6 +91,7 @@
       end with "/", the directory itself with all contents is copied.
     - If path is a file and dest ends with "\", the file is copied to the
       folder with the same filename.
+    - Not required if using C(content)
```
Comment (samdoran) [inline lib/ansible/modules/windows/win_copy.py]: This seems to contradict `required: yes`. If this is not always required, also change to `required: no` and note cases in which it is required.

Comment (bcoca) [inline lib/ansible/modules/windows/win_copy.py]: remove `required: yes`, if this is the case, that is incorrect documentation, make note elsewhere that 'either content or src is required', but each one should be `required: no` themselves.

Comment (acozine) [reply lib/ansible/modules/windows/win_copy.py]: Nice catch @samdoran. @ShachafGoldstein if this parameter is not required in all cases, remove the `required: yes` line - we only add a line for `required` when `required: yes`.

Comment (acozine) [reply lib/ansible/modules/windows/win_copy.py]: Maybe the new line 94 could be `- Required unless using C(content).`

**Pull request merged**